compiler/rustc_borrowck/src/region_infer/opaque_types.rs

```diff
@@ -275,11 +293,13 @@ impl<'tcx> RegionInferenceContext<'tcx> {
                     None => {
                         // Nothing exact found, so we pick the first one that we find.
                         let scc = self.constraint_sccs.scc(vid);
-                        for vid in self.rev_scc_graph.as_ref().unwrap().upper_bounds(scc) {
-                            match self.definitions[vid].external_name {
-                                None => {}
-                                Some(region) if region.is_static() => {}
-                                Some(region) => return region,
+                        if let Some(rev_scc_graph) = &self.rev_scc_graph {
```

Comment:
when is the `rev_scc_graph` `None` here?

Reply:
Resolved by removing the relevant commits now that https://github.com/rust-lang/rust/pull/116891 has landed